tests/bud.bats

```diff
@@ -6325,10 +6327,10 @@ _EOF
   expect_output --substring "world"
 }
 
-@test "bud-verify-if-we-dont-clean-prexisting-path" {
+@test "bud-verify-if-we-dont-clean-preexisting-path" {
   skip_if_no_runtime
   skip_if_in_container
-  _prefetch alpine debian
+  _prefetch alpine ubuntu
```

Comment:
eliminate unused image (for all purposes here, the ubuntu image works the same as debian)